active-rfcs/0000-tables.md

```diff
@@ -0,0 +1,238 @@
+- Start Date: 2024-07-13
+- Referenced Issues: https://github.com/excalidraw/excalidraw/issues/4847
+- Implementation PR: (leave this empty)
+
+# Summary
+
+Once this feature is implemented the users will be able to use tables in excalidraw. The `table` will be added to the shapes toolbar similar to other tools.
```

Comment:
Since the toolbar is too big now, I am afraid that we will not be able to add in the toolbar unless we move some shapes inside a dropdown.
@dwelle any thoughts here on where should the table icon be added ?